apps/consulting/services/posts/getLibraryTypes/getLibraryTypes.ts

```diff
@@ -0,0 +1,25 @@
+import { isValidArray, isValidObject } from '@quansight/shared/utils';
+
+import { getDataSourceEntries } from '../../../api/utils/getDataSourceEntries';
+import { TLibraryFilter } from '../../../types/utils/LibraryFilter';
+
+export const getLibraryTypes = async (
```

Comment:
This looks like it doesn't inject `blogs` as a LibraryType.

Does this mean that, even though the blogs will now live in Github/Markdown, we still need to retain the `'blog': 'blog'` item in the `Post Type` datasource in Storyblok?

<img width="700" alt="image" src="https://user-images.githubusercontent.com/11325439/204393383-f11bf992-aa96-47bc-b3b6-44fe7d787d12.png">

Reply:
Fixed, now `blog` type will be injected in `getLibraryTypes`.